### ISSUE TITLE
0.5.2 class file for IRiemannClient not found

### ISSUE THREAD

**Issue by mbtts** [riemann/riemann-java-client#99]
Hi,

I am seeing an error with this class after updating to `0.5.2` (`0.5.1` works as expected).

Source to replicate: https://github.com/mbtts/metrics4-riemann-reporter-compile-error.

```java
package com.example;

import com.codahale.metrics.riemann.Riemann;
import io.riemann.riemann.client.IRiemannClient;

public class RiemannExtended extends Riemann {
    public RiemannExtended(IRiemannClient client) {
        super(client);
    }
}
```

The error is:
```
> Task :compileJava FAILED
…RiemannExtended.java:8: error: cannot access IRiemannClient
        super(client);
             ^
  class file for IRiemannClient not found
1 error

FAILURE: Build failed with an exception.
```

I have attempted using a variety of JDKs, but all produce same result. My IDE indicates the bytecode does not match the sources and [fernflower](https://github.com/fesh0r/fernflower) is unable to decompile:

```java
//
// Source code recreated from a .class file by IntelliJ IDEA
// (powered by FernFlower decompiler)
//

package com.codahale.metrics.riemann;

import java.io.Closeable;
import java.io.IOException;

public class Riemann implements Closeable {
    IRiemannClient client;

    public Riemann(String var1, Integer var2) throws IOException {
        throw new Error("Unresolved compilation problems: \n\tThe import io cannot be resolved\n\tThe import io cannot be resolved\n\tThe import io cannot be resolved\n\tThe import io cannot be resolved\n\tIRiemannClient cannot be resolved to a type\n\tThe method getClient(String, Integer, int) from the type Riemann refers to the missing type IRiemannClient\n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n\tRiemannClient cannot be resolved\n\tRiemannBatchClient cannot be resolved to a type\n\tUnsupportedJVMException cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n");
    }

    public Riemann(String var1, Integer var2, int var3) throws IOException {
        throw new Error("Unresolved compilation problems: \n\tThe import io cannot be resolved\n\tThe import io cannot be resolved\n\tThe import io cannot be resolved\n\tThe import io cannot be resolved\n\tIRiemannClient cannot be resolved to a type\n\tThe method getClient(String, Integer, int) from the type Riemann refers to the missing type IRiemannClient\n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n\tRiemannClient cannot be resolved\n\tRiemannBatchClient cannot be resolved to a type\n\tUnsupportedJVMException cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n");
    }

    private static IRiemannClient getClient(String var0, Integer var1, int var2) throws IOException {
        throw new Error("Unresolved compilation problems: \n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n\tRiemannClient cannot be resolved\n\tRiemannBatchClient cannot be resolved to a type\n\tUnsupportedJVMException cannot be resolved to a type\n");
    }

    public Riemann(IRiemannClient var1) {
        throw new Error("Unresolved compilation problems: \n\tThe import io cannot be resolved\n\tThe import io cannot be resolved\n\tThe import io cannot be resolved\n\tThe import io cannot be resolved\n\tIRiemannClient cannot be resolved to a type\n\tThe method getClient(String, Integer, int) from the type Riemann refers to the missing type IRiemannClient\n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n");
    }

    public void connect() throws IOException {
        throw new Error("Unresolved compilation problems: \n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n");
    }

    public void close() throws IOException {
        throw new Error("Unresolved compilation problems: \n\tIRiemannClient cannot be resolved to a type\n\tIRiemannClient cannot be resolved to a type\n");
    }
}
```

Any thoughts/suggestions would be appreciated.

**Comment by jamtur01** [comment]
Okay - this is odd. Can you try:

File -> Invalidate Caches/ Restart 

In IntelliJ and try again?

**Comment by jamtur01** [comment]
Okay - metrics3 works - so I am confused as to what's happening here.

**Comment by jamtur01** [comment]
Okay - it looks like a change in mvn build process has caused an issue. I need to poke at it.

**Comment by jamtur01** [comment]
This should be fixed in 0.5.3. I tested it using local jars using your test case and it appears to be good. Let me know if you have any issues.

**Comment by mbtts** [comment]
Thats fantastic, thank you very much for the fast turnaround. 

Greatly appreciated ☺️
